forms-flow-components/src/components/SvgIcons/index.tsx

```diff
@@ -635,3 +634,22 @@ export const TickIcon = ({ color = baseColor, ...props }) => (
    strokeLinejoin="round"/>
 </svg>
 );
+
+export const DropdownIcon = () => (
+  <svg
+    xmlns="http://www.w3.org/2000/svg"
+    width="16"
+    height="9"
+    viewBox="0 0 16 9"
+    fill="none"
+  >
+    <path
+      d="M1.5 1.24951L8.00058 7.7501L14.5006 1.25009"
+      stroke="#253DF4"
```

Comment:
pls add color as props with baseColor as default. then we can replace stroke="#253DF4" with stroke={color}